src/sentry/api/endpoints/team_members.py

```diff
@@ -3,23 +3,54 @@
 from rest_framework.response import Response
 
 from sentry.api.bases.team import TeamEndpoint
-from sentry.api.serializers import serialize
-from sentry.models import InviteStatus, OrganizationMember
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import InviteStatus
+from sentry.models.organizationmemberteam import OrganizationMemberTeam
+
+
+@register(OrganizationMemberTeam)
+class DetailedOrganizationMemberTeamSerializer(Serializer):
+    def __init__(self, *args, **kwargs):
+        self.team = kwargs.pop("team", None)
+        super().__init__(*args, **kwargs)
+
+    def get_attrs(self, item_list, user, **kwargs):
+        attrs = {}
+        for org_member_team in item_list:
+            attrs[org_member_team] = {
+                "org_member": org_member_team.organizationmember,
+                "user": org_member_team.organizationmember.user,
+            }
+        return attrs
```

Comment:
`get_attrs` is unnecessary here since you're not performing any bulk querying. However, I'd generally recommend using `prefetch_related_objects(item_list, "organizationmember", "organizationmember__user")` in here, rather than relying on the caller to remember to use `prefetch_related` or `select_related`